lib/sass_spec/stats/todo.rb

```diff
@@ -0,0 +1,240 @@
+    end
+
+    parser.on("-h", "--help", "Print this help message.") do
+      puts parser.help()
+      return nil
+    end
+
+    parser.parse!(args)
+    new(options)
+  end
+
+  def initialize(options)
+    @options = options
+  end
+
+  def stats
+    files = Dir.glob(File.join(SassSpec::SPEC_DIR, '**/*')).uniq
+
+    if @options[:impl]
+      todos_by_implementation(files, @options[:impl])
+    elsif @options[:issue]
+      todos_by_issue(files, @options[:issue])
+    elsif @options[:missing_issue]
+      todos_missing_issue(files)
+    else
+      todos(files)
+    end
+  end
+
+  def todos_by_implementation(files, implementation)
+    regexes = []
+    if implementation == "dart-sass"
+      dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/dart-sass#\d+$', Regexp::MULTILINE)
+      dart_sass_missing_issue = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+      regexes.push(dart_sass, dart_sass_missing_issue)
+    elsif implementation == "libsass"
+      libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/libsass#\d+$', Regexp::MULTILINE)
+      libsass_missing_issue = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+      regexes.push(libsass, libsass_missing_issue)
+    end
+
+    todos = find_todos(files, regexes)
+    print_report(todos)
+  end
+
+  def todos_by_issue(files, issue)
+    if issue =~ /dart-sass/
+      regex = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/' + issue + '\d+$', Regexp::MULTILINE)
+    elsif issue =~ /libsass/
+      regex = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/' + issue + '$', Regexp::MULTILINE)
+    end
+
+    todos = find_todos(files, [regex])
+    print_report(todos)
+  end
+
+  def todos_missing_issue(files)
+    dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+    libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+
+    todos = find_todos(files, [dart_sass, libsass])
+    print_report(todos)
+  end
+
+  def todos(files)
+    dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/dart-sass#\d+$', Regexp::MULTILINE)
+    dart_sass_missing_issue = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+    libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/libsass#\d+$', Regexp::MULTILINE)
+    libsass_missing_issue = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+
+    todos = find_todos(files, [
+      dart_sass,
+      dart_sass_missing_issue,
+      libsass,
+      libsass_missing_issue,
+    ])
+    print_report(todos)
+  end
+
+  def find_todos(files, regexes)
+    todos = {}
+
+    files.each do |file|
```

Comment:
I'd recommend using `SassSpec::Directory#glob` here to find all the `options.yml` files. It already has built-in logic for normalizing away the differences between files in HRX and physical files.

Reply:
Thanks for pointing me to this!